lib/shell.js

```diff
@@ -34,6 +34,8 @@ module.exports = function(cmd, args, options, callback) {
         }
       };
 
+  process.setMaxListeners(0);
```

Comment:
It seemed like this was happening because I had more than whatever the max was (I think 10) shell commands running at once.